src/App.js

```diff
@@ -1,21 +1,73 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { Route, Redirect } from 'react-router-dom'
+import logo from './logo.svg'
+import './App.css'
+import { Tabs } from 'antd'
+import { createLocation } from 'history'
```

Comment:
I'd rather keep the usage of `;` consistent everywhere, so (for now?) could you please put them back here?